mpt/core/src/main/java/org/apache/james/mpt/api/ImapFeatures.java

```diff
@@ -28,7 +28,10 @@
 public class ImapFeatures {
     
     public enum Feature {
-        NAMESPACE_SUPPORT
```

Comment:
Antoine, I have a commit for your last request, but I don't like it : 
 - I encounter difficulties with Quota support, which depends on wether Quota stuff was setted or not (mutable by definition) while trying to do something immutable (move ImapFeatures to mailbox/api, renamed MailboxFeature)
 - I have one more concern : for Cyrus, ImapFeature is more suited than MailboxFeature